docs/runbooks/HPPSharingPoolPathWithOS.md

```diff
@@ -0,0 +1,30 @@
+# HPPSharingPoolPathWithOS
+
+## Meaning
+
+The hostpath-provisioner (HPP) is used to dynamically provision hostPath volumes to provide storage for PVCs.  
+If HPP is sharing a filesystem with other critical components (kubelet / OS) its possible that HPP PVs will cause node disk pressure.
+## Impact
+
+Possible degraded node operation.
+
+## Diagnosis
+
+- Check which HPP pool is being shared with OS by looking at DaemonSet logs
+	```bash
+	export HPP_NAMESPACE="$(kubectl get deployment -A | grep hostpath-provisioner-operator | awk '{print $1}')"
+	```
+
+- Find CSI DaemonSet pods.
+	```bash
+	kubectl -n $HPP_NAMESPACE get pods | grep hostpath-provisioner-csi
+	```
+ 
+- Check pod logs for "pool <pool_name>, <pool_path>, shares path with OS which can lead to node disk pressure".
```

Comment:
@akalenyu I didnt understand this line. Is <pool_name>, <pool_path> supposed to be the ones that are mention in the alert.
If yes, perhaps write:
Check pod logs for the pool_name and pool_path mentioned in the alert. Replace the <csi_daemonset_pod> with the pool_name in the below command.